src/main/java/de/tum/in/www1/artemis/web/rest/ManagementResource.java

```diff
@@ -0,0 +1,45 @@
+package de.tum.in.www1.artemis.web.rest;
+
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import de.tum.in.www1.artemis.service.feature.Feature;
+import de.tum.in.www1.artemis.service.feature.FeatureToggleService;
+
+@RestController
+@RequestMapping(ManagementResource.ROOT_MANAGEMENT)
+public class ManagementResource {
+
+    static final String ROOT_MANAGEMENT = "/api/management";
+
+    private static final String SUB_FEATURE_TOGGLE = "/feature-toggle";
```

Comment:
Any Advantage of doing this instead of using the strings in the Annotations?

Reply:
- You have an overview over all endpoints right when opening the file
- It's much easier to change the endpoint URL if you just have to look at the top of the related controller
- If you choose to do so, you can actually easier refactor this later on to using value injection.

Reply:
But are we doing this anywhere else?

Reply:
No, but since this was a new resource I thought I'd use the opportunity to start doing this.

Reply:
ok, let's add this as a discussion point to our meeting on Tuesday and discuss in the team. 👍 
For now let's keep it as it is!